src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -762,3 +764,5 @@ def separate_out_esm_packages(self, packages, package_versions):
         self.composite_logger.log_debug("Filter esm packages : [TotalPackagesCount={0}][EsmPackagesCount={1}]".format(len(packages), len(ua_esm_required_packages)))
         return non_esm_packages, non_esm_package_versions, ua_esm_required_packages, ua_esm_required_package_versions, ua_esm_required_packages_found
 
+    def get_package_install_expected_avg_time_in_minutes(self):
+        return self.package_install_expected_avg_time_in_minutes
```

Comment:
Leave 2 empty lines at the end of each file. See others for reference

Reply:
Addressed it.

Reply:
2 empty lines

Reply:
2 empty lines

Reply:
I have added 2 empty lines in all the files. I can see that in my local repo. I also tried to checkout the branch garamrak-multiPhase in another machine and can see the empty lines there also. Not sure why it is not visible in the PR.
Can you please verify that there are empty lines added after checkout to the garamrak-multiPhase branch